lib/LanguageServer/Util/ClientCapabilitiesProvider.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServer\Util;
+
+use InvalidArgumentException;
+use Phpactor\Extension\LanguageServer\Helper\ArrayHelper;
+use Psr\Container\ContainerInterface;
+
+final class ClientCapabilitiesProvider implements ContainerInterface
```

Comment:
Why implement containerInterface?

Reply:
At first it looks like one, but I reverted my choice and forgot the interface.
I will remove it.